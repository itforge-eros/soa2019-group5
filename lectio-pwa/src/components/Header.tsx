import React, { Component } from 'react';
import styles from './Header.module.sass';

class Header extends Component {
	render() {
		return (
			<div className={styles.header}>
				{this.props.children}
			</div>
		)
	}
}

export default Header;
