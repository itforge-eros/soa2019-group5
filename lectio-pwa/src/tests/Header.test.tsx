import React from 'react';
import renderer from 'react-test-renderer';
import Header from "../components/Header";

it('renders Header successfully', () => {
	const tree = renderer
		.create(
			<Header/>
		)
		.toJSON();
	expect(tree).toMatchSnapshot();
});

it('renders Header with child successfully', () => {
	const tree = renderer
		.create(
			<Header>Hello</Header>
		)
		.toJSON();
	expect(tree).toMatchSnapshot();
});

export default undefined
