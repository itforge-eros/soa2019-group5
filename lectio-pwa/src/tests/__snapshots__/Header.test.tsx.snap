// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header successfully 1`] = `
<div
  className="header"
/>
`;

exports[`renders Header with child successfully 1`] = `
<div
  className="header"
>
  Hello
</div>
`;
